app/pods/components/input/md-date-range/component.js

```diff
@@ -20,7 +20,8 @@ const Validations = buildValidations({
     validator('presence', {
       presence: true,
       disabled: notEmpty('model.end'),
-      ignoreBlank: true
+      ignoreBlank: true,
+      message: 'Start or end date is required',
```

Comment:
Since the messages are the same, could use a _global option_ here. See http://offirgolan.github.io/ember-cp-validations/docs/modules/Advanced.html#global-options

Reply:
@jlblcc, I think this is the correct implementation:
```
start: [
    validator('presence', {
      presence: true,
      disabled: notEmpty('model.end'),
    })
  ],
  end: {
    validators: [
      validator('date', {
        onOrAFter: alias('model.start'),
        isWarning: true
      }),
      validator('presence', {
        presence: true,
        disabled: notEmpty('model.start'),
      } )
    ]
  }
}, {
  message: 'Start or end date/time is required',
  ignoreBlank: true
});
```
![image](https://user-images.githubusercontent.com/24277002/78911726-b32af580-7a54-11ea-9ca8-080f3d8aa3a1.png)